R/wrangle_and_check.R

```diff
@@ -80,16 +80,7 @@ wrangle_and_check_pacta_results <- function(pacta_results, start_year, time_hori
       year = seq(start_year, start_year + time_horizon),
       tidyr::nesting(!!!rlang::syms(nesting_vars_lookup))
     ) %>%
-    dplyr::mutate(plan_tech_prod = dplyr::if_else(is.na(.data$plan_tech_prod), 0, .data$plan_tech_prod)) %>%
-    apply_filters(
-      investor = investor_name_placeholder,
-      sectors = sectors_lookup,
-      technologies = technologies_lookup,
-      scenario_geography_filter = scenario_geography_filter,
-      scenarios = scenarios_filter,
-      allocation_method = allocation_method,
-      start_analysis = start_year
-    )
+    dplyr::mutate(plan_tech_prod = dplyr::if_else(is.na(.data$plan_tech_prod), 0, .data$plan_tech_prod))
```

Comment:
I surfaced this mutate from the previous apply_filters, all actual filters were moved to the processing function (they were mostly redundant)

Reply:
I compared pacta_results between master and PR version for type bond and see that PR version has quite a bunch of more rows (master: 7836, PR: 9552).

Reply:
compare pacta_results_new %>% dplyr::anti_join(pacta_results_old) %>% View(), I can not see a clear pattern, so is this expected?

Reply:
ok